From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Alice Zhao <alice@makenotion.com>
Date: Thu, 20 Jun 2024 14:50:13 -0700
Subject: test: patch CRLF test file for allcan_read_write patch

Refs https://chromium-review.googlesource.com/c/v8/v8/+/5006387
Should be upstreamed.

This patch is seperated from the apply_allcan_read_write.patch
because the file itself has CRLF line endings and need to be patched
seperately. When this patch is removed, also remove the eol=crlf
exception in .gitattributes.

diff --git a/test/js/methodswithdata-test.js b/test/js/methodswithdata-test.js
index 9f4fc5246f1ca3d8faadc9a72d49e14a6a190e32..795538510431fb3b633be37f916d4ce1955c808a 100644
--- a/test/js/methodswithdata-test.js
+++ b/test/js/methodswithdata-test.js
@@ -16,7 +16,7 @@ test('SetMethod with data', function (t) {
 });
 
 test('accessors with data', function (t) {
-    t.plan(7)
+    t.plan(6)
     var settergetter = bindings.create()
     t.equal(settergetter.prop1, 'this is property 1')
     t.ok(settergetter.prop2 === '')
@@ -33,6 +33,5 @@ test('accessors with data', function (t) {
     t.equal(derived.prop1, 'this is property 1')
     derived.prop2 = 'setting a new value'
     t.equal(derived.prop2, 'setting a new value')
-    t.equal(settergetter.prop2, 'setting a new value')
   })
   
\ No newline at end of file
